.github/pull_request_template.md

```diff
@@ -1,48 +1,59 @@
+
+NOTE: Please grant permission for repository maintainers to edit your PR.  It is EXTREMELY common for PRs to be held up due to trivial changes being requested and the author being unavailable to make them.
 -->
 
 #### Summary
 <!--
-A one-line description of your change that will be extracted and added to the [project changelog](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/changelog.txt).
+This section should consist of exactly one line, formatted like this:
+
+`SUMMARY: [Category] "[Briefly describe the change]"`
+
+The backquotes `` are optional, and you should not enter the square brackets [].  Category must be one of these:
 
-The format is (ignore the square brackets): ```SUMMARY: [Category] "[description]"```
+- Features
+- Content
+- Interface
+- Mods
+- Balance
+- Bugfixes
+- Performance
+- Infrastructure
+- Build
+- I18N
 
-The categories to choose from are:
+For more on the meaning of each category, see:
+https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CHANGELOG_GUIDELINES.md
 
-* Features
-* Content
-* Interface
-* Mods
-* Balance
-* Bugfixes
-* Performance
-* Infrastructure
-* Build
-* I18N
+If approved and merged, your summary will be added to the project changelog:
+https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/changelog.txt
 
-Example: ```SUMMARY: Content "Adds new mutation category 'Mouse'"```
+Example `SUMMARY` lines: (only enter ONE!)
 
-See the [Changelog Guidelines](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CHANGELOG_GUIDELINES.md) for explanations of the categories.
+SUMMARY: Features "Implement item pockets"
+SUMMARY: Content "Add nested maps to 4 basements"
+SUMMARY: Bugfixes "Fix bugs and increase performance of 3D shadowcasting"
+SUMMARY: Infrastructure "Improve test suite for item info panel"
 -->
 
 #### Purpose of change
 <!--
-If there's an existing issue describing the problem this PR addresses or the feature it adds, please link it like: ```#1234```
-If it *fully* resolves an issue, link it like: Fixes #1234
-Even if the issue describes the problem, please provide a few-sentence summary here.
-Example: Fixes #1234 - XL mutants cannot wear arm/leg splints due to missing OVERSIZE flag.
+With a few sentences, describe your reasons for making this change.  If it relates to an existing issue, you can link it with a `#` followed by the Github issue number, like `#1234`.
+If your pull request *fully* resolves an issue, include the word `Fix` or `Fixes` before the issue number, like: `Fixes #1234`.
+
+Example:
+
+Fixes #1234 - XL mutants cannot wear arm/leg splints due to missing OVERSIZE flag.
+
 If there is no related issue, please describe the issue you are addressing, including how to trigger a bug if this is a bugfix.
 Don't put the backticks around the `#` and issue or pull request number to allow the GitHub automatically reference to it.
```

Comment:
Since the examples are already commented out by `<!-- -->` we can probably remove this line and remove backticks surrounding the examples.

Reply:
I agree - this is cleaned up in the latest version.